docs/tutorials/flow-task-config.md

```diff
@@ -80,6 +80,20 @@ def my_flow():
 
 You don't have to supply a version for your flow. By default, Prefect makes a best effort to compute a stable hash of the `.py` file in which the flow is defined to automatically detect when your code changes.  However, this computation is not always possible and so, depending on your setup, you may see that your flow has a version of `None`.
 
+You can also distinguish runs of this flow by providing a `flow_run_name`; this setting accepts a string that can optionally contain templated references to the parameters of your flow. The name will be formatted using Python's standard string formatting syntax as can be seen here:
```

Comment:
Thank you for updating the documentation!

@tpdorsey do we want blurbs in the flow/task concept docs?

Reply:
Yes! I think even copying this verbatim to the Flows concept docs would be fine, with "You can also distinguish runs of a flow...". Likewise for the task example to Tasks.

Reply:
Updated!